doc/rados/operations/control.rst

```diff
@@ -56,14 +56,14 @@ To list the cluster's keys and their capabilities, execute the following::
 Placement Group Subsystem
 =========================
 
-To display the statistics for all placement groups, execute the following:: 
+To display the statistics for all placement groups, execute the following::
```

Comment:
@chengweiv5, does this fix a formatting problem in the output or silence a warning of sphinx?